src/libcollections/slice.rs

```diff
@@ -691,15 +691,35 @@ impl<T> [T] {
     ///
     /// # Examples
     ///
-    /// Print the slice split by numbers divisible by 3 (i.e. `[10, 40]`,
-    /// `[20]`, `[50]`):
+    /// ```
+    /// let slice = [10, 40, 33, 20];
+    /// let mut iter = slice.split(|num| num % 3 == 0);
+    /// assert_eq!(iter.next().unwrap(), &[10, 40]);
+    /// assert_eq!(iter.next().unwrap(), &[20]);
+    /// assert!(iter.next().is_none());
+    /// ```
+    ///
+    /// If the first or last element in the slice is matched, an empty slice
```

Comment:
I'm not sure to understand the last part of the sentence "an empty slice will be returned first or last respectively by the iterator.". An empty slice will be returned first or last?